index.html

```diff
@@ -3,7 +3,9 @@
 
 <head>
     <title>US Weather APP</title>
+    <!-- Any reason why you chose an initial-scale less than 1? -->
```

Comment:
Well ... This was a pre-coffee decision based solely on how it looked on my phone. Not really a thought out choice, but more of an afterthought when i was trying to push this up.